apps/vanilla/src/pages/exercise2/scripts/DiceResults.ts

```diff
@@ -0,0 +1,21 @@
+import { DiceAccumulator } from './DiceAccumulator';
+import { ISubscriber, TurnResult } from './types';
+
+/** Common dice results. */
+export class DiceResults implements ISubscriber<TurnResult> {
+
+	/** Turns results. */
+	public readonly results: DiceAccumulator;
+
+	public constructor() {
+		this.results = new DiceAccumulator();
+	}
```

Comment:
You can omit constructor in this case:

```suggestion
	/** Turns results. */
	public readonly results = new DiceAccumulator();;
```